addons/binding/org.openhab.binding.tadoac/README.md

```diff
@@ -0,0 +1,61 @@
+# TadoAC Binding
+
+This is the TadoAC binding which uses the unofficial v2 REST-API.  
+
+## Supported Things
+
+Currently the binding supports the Tado AC Control (IR). 
+
+## Discovery
+
+Tado uses a cloud service for their devices. So discovery is not available
```

Comment:
Just out of curiosity: Did you have a look on the (binary) protocol of the AC device itself? I.e. had a look in wireshark to see if you could send commands directly to the AC without using their cloud service?